cypress/integration/unit_tests/click_feature_test.js

```diff
@@ -94,7 +84,8 @@ describe('Unit tests for click_feature.js with map and table', () => {
       tableDiv.id = 'table';
       containerDiv.appendChild(tableDiv);
       drawTableAndSetUpHandlers(
-          convertEeObjectToPromise(scoredFeatures).then((fc) => fc.features),
+          EePromiseCache.convertEeObjectToPromise(scoredFeatures)
```

Comment:
why not just regular import here? were you stubbing something before?

Reply:
Oops, good guess, I was. Removed.